lib/beetle/client.rb

```diff
@@ -87,16 +87,16 @@ def register_exchange(name, options={})
     #   the binding key (defaults to the name of the queue)
     # [<tt>:lazy</tt>]
     #   whether the queue should use lazy mode (defaults to <tt>config.lazy_queues_enabled</tt>)
-    # [<tt>:dead_lettering</tt>]
-    #   whether the queue should use dead lettering (defaults to <tt>config.dead_lettering_enabled</tt>)
+    # [<tt>:queue_properties</tt>]
+    #   whether the queue should use dead lettering (defaults to <tt>config.queue_properties_enabled</tt>)
```

Comment:
The option name and description don't fit together.

Reply:
fixed